ic-os/components/ic/generate-ic-config/generate-ic-config.sh

```diff
@@ -228,12 +228,16 @@ QUERY_STATS_EPOCH_LENGTH="${query_stats_epoch_length:-600}"
 # TODO: If the Jaeger address is not specified the config file will contain Some(""). This needs to be fixed.
 JAEGER_ADDR="${jaeger_addr:-}"
 
+# TODO: Should pass prefix directly
+IPV6_PREFIX=$(echo "${IPV6_ADDRESS}" | sed -E -e 's/:/#/4' -e 's/#.*/::\/64/')
+
```

Comment:
Currently in the new Config, GuestOS holds a `FixedIpv6Config` object:
```
pub struct FixedIpv6Config {
    // Fixed ipv6 address includes subnet mask /64
    pub address: String,
    pub gateway: Ipv6Addr,
}
```

Should I add ipv6_prefix to it?

Reply:
Maybe, we can leave this for now, and we can put it on the pile with the rest of the network revamp changes.

Reply:
Sounds good. So right now it will always default to using the loopback address and later we'll pass the prefix to the Guest.

Reply:
The loopback is just a safety in case the calculation from address doesn't work out. It would be nice to eventually use the prefix directly (since it does exist directly at one point) or build this out more so that we are using proper network tools over `sed`.

Reply:
Oh I see! I misread 👍